_posts/2020-05-20-gsoc-students.md

```diff
@@ -0,0 +1,124 @@
+
+<img
+src="https://cbouy.github.io/assets/img/photo-CV.jpeg"
+title="Cédric Bouysset" alt="Cédric Bouysset"
+style="float: left; width: 110px; height: 110px; border-radius: 20px; border: 15px solid white" />
+
+The aim of the [RDKit interoperability
+project](https://summerofcode.withgoogle.com/projects/#6750913248624640)
+is to give MDAnalysis the ability to use [RDKit][]’s `Chem.Mol`
+structure as an input to an MDAnalysis’ `Universe`, but also to
+convert a `Universe` or `AtomGroup` to an RDKit `molecule`. RDKit is
+one of the most complete and one of the most commonly used
+chemoinformatics package, yet it lacks file readers for formats
+typically encountered in MD simulations. @cbouy will implement in
+MDAnalysis the ability to switch back and forth between a `Universe`
+and an RDKit `molecule` to perform typical chemoinformatics
+calculations and so add a lot of value to both packages.
+
+Cédric is a PhD student at the Institute of Chemistry of Nice, in the
+south of France. His research project tackles taste perception (and
+sometimes olfaction) using computational approaches. His day-to-day
+work includes homology modeling of bitter taste receptors, building
+machine-learning models to predict taste modalities from chemical
+structures, writing Python or Bash scripts, and a bit of teaching.
+
+Cédric will describe his progress in [his blog][cbouy-blog].
+
+
+## Yuxuan Zhuang: Serialize Universes for parallel
+
+<img
+src="https://i0.wp.com/www.biophysics.se/wp-content/uploads/2018/06/IMG_4767.jpg"
+title="Yuxuan Zhuang" alt="Yuxuan Zhuang"
+style="float: left; width: 110px; height: 110px; border-radius: 20px; border: 15px solid white" />
+
+MDAnalysis is a flexible and relatively fast framework for complex
+analysis tasks in molecular dynamics (MD) simulations. To achieve a
+flawless implementation of parallelism, @yuxuanzhuang will implement
+[serialization support for
+`Universe`](https://summerofcode.withgoogle.com/projects/#5812065073102848),
+the core of MDAnalysis. Furthermore, he will adapt this new
+serialization functionality into Dask, multiprocessing, or
+MPI. Additionally, he will run tests, write documentation, and run
+benchmarks.
+
+Yuxuan is a first-year PhD student at Stockholm University. He mainly
+works on understanding ion channels from MD simulations in Prof. Erik
+Lindahl’s group. My daily workflow involves setting up simulations,
+running simulations on our own clusters or the HPC center, and
+performing various analyses on the trajectories in my jupyter
+notebook.
+
+Yuxuan will chronicle his work on [his blog
+"Simulacrum"][yuxuanzhuang-blog].
+
+
+— @richardjgowers @IAlibay @acmnpv @fiona-naughton @orbeckst (mentors)
+
+[gsoc]: https://summerofcode.withgoogle.com
+[mda-gsoc]: https://summerofcode.withgoogle.com/organizations/4891814374408192/
+[Gromacs]: http://www.gromacs.org
+[RDKit]: http://rdkit.org/
+[hmacdope-gsoc]: https://summerofcode.withgoogle.com/projects/
```

Comment:
The gsoc entries here don't seem to be used by anything. Are they worth keeping?